### PR TITLE
Update Lesson_15_Display_NFTs_From_Candy_Machine.md

### PR DESCRIPTION
**Pull request by shungo0222**
Fixed some typos.

### CODE REVIEW

File: Solana_Core/en/Core_2/Lesson_15_Display_NFTs_From_Candy_Machine.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const FetchCandyMachine: FC = () => {
     // fetch candymachine data
     try {
       const candyMachine = await metaplex
-        .candyMachinesV2()
+        .candyMachines()
```
Comment (helloitsm3) [inline Solana_Core/en/Core_2/Lesson_15_Display_NFTs_From_Candy_Machine.md:64]: Have you tried running this to make sure it works using the latest Candy machine cli?

Comment (shungo0222) [reply Solana_Core/en/Core_2/Lesson_15_Display_NFTs_From_Candy_Machine.md:64]: Oh sorry, my bad. I was using an older version.

**Comment by shungo0222** [comment]
I undid. `candyMachines()` -> `candyMachinesV2()`
And removed `run()` because you don't need it in the latest version.
https://metaplex-foundation.github.io/js/classes/js.CandyMachinesV2Client.html#findByAddress

**Review by helloitsm3 — APPROVED** [review]
Thank you so much for this

**Pull request merged**